make searches for -lmoo targets in /lib and /usr/lib after mangling
them with .LIBPATTERNS into libmoo.so

This patch extends to teh search path to /lib64/ and /usr/lib64

--- remake.c	2001/12/11 15:59:03	1.1
+++ remake.c	2001/12/11 16:01:01
@@ -1252,6 +1252,8 @@
   static char *dirs[] =
     {
 #ifndef _AMIGA
+      "/lib64",
+      "/usr/lib64",
       "/lib",
       "/usr/lib",
 #endif
